BIPs/2023-W22/BIP-316b.json

```diff
@@ -0,0 +1,32 @@
+{
+  "version": "1.0",
+  "chainId": "1",
+  "createdAt": 1685637126103,
+  "meta": {
+    "name": "Transactions Batch",
+    "description": "",
+    "txBuilderVersion": "1.14.1",
+    "createdFromSafeAddress": "0xc38c5f97B34E175FFd35407fc91a937300E33860",
+    "createdFromOwnerAddress": "",
+    "checksum": "0x0ac587541b6f87123cb0605cdd54f7e2e0c1c8fc3ad28432262ae655ae65e2b4"
+  },
+  "transactions": [
```

Comment:
The Balancer Maxi LM Multisig eth:0xc38c5f97B34E175FFd35407fc91a937300E33860 will interact with the GaugeAdderv4 at 0x5DbAd78818D4c8958EfF2d5b95b28385A22113Cd and call the addGauge function with the following argument:

gauge: 0xb78543e00712C3ABBA10D0852f6E38FDE2AaBA4d which corresponds to the new 10% capped gauge for veBAL

gaugeType: EthereumSingleRecipientGauge